src/encoder.rs

```diff
@@ -1729,19 +1729,19 @@ fn aom_uleb_encode(mut value: u64, coded_value: &mut [u8]) -> usize {
   leb_size
 }
 
+pub fn write_temporal_delimiter(
+  packet: &mut dyn io::Write
+) -> io::Result<()> {
+  packet.write(&[0x12, 0x00])?;
```

Comment:
Can you use the OBU_Type::OBU_TEMPORAL_DELIMITER enum?

Reply:
Yes, but not for the second byte which is the length.